src/nbsap/utils.py

```diff
@@ -42,3 +43,42 @@ def generate_code(model, instance):
         code = '{0}'.format(int(last_code) + 1)
 
     return code
+
+
+def sort_by_code(value):
+    try:
+        return natsorted(value, key=lambda i: i.code.split('.'))
```

Comment:
Why are you splitting by `.`? How will the result of this line be different from the one on line 52?